main.ts

```diff
@@ -2,19 +2,17 @@ import { app, BrowserWindow, screen, Menu, nativeTheme, MenuItemConstructorOptio
 import * as path from 'path';
 import * as url from 'url';
 import { createMenuOptions } from './electron-utils/menu-bar';
+import { isDeveloperMode, isLinuxOs, isMacOs, isWindowsOs, appTitle } from './electron-utils/supporting-logic';
 import { getAccessToken } from './oauth';
 
 const Logger = require('electron-log');
 const ICON_PATH = path.join(__dirname, 'dist/favicon.512x512.png');
 
 let win: BrowserWindow = null;
-const args = process.argv.slice(1),
-  serve = args.some(val => val === '--serve');
+const isDevMode = isDeveloperMode();
```

Comment:
I feel that it might not be necessary to have this `constant` ><

We can just pass the return result of the function into the if statement like this 

```typescript
if (isDeveloperMode()) {
  // ....
}

Not sure what do yall think! >< @anubh-v @kkangs0226 
```

Reply:
I'm not entirely sure of what the best practice would be but I think its fine since @kkangs0226 references it once again below when deciding to load the localhost or the `index.html` from `/dist`.

Also,

> We can just pass the return result of the function into the if statement like this 
>
> ```typescript
> if (isDeveloperMode()) {
>   // ....
> }
> ```

Given that the first use is to pass a `synchronous-message` to a different component within angular, it wouldn't be "clean" to do an `if-else` imo.